<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllReservationsWindow</class>
 <widget class="QWidget" name="AllReservationsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="closeButtonLayout">
     <item alignment="">
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#closeButton {
    background-color: #ff3131;
    border: none;
    border-radius: 5;
    font-size: 14px;
}

#closeButton:hover {
    background-color: #be2222;
}
</string>
       </property>
       <property name="text">
        <string>Cofnij</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mainLabelLayout">
     <item>
      <widget class="QLabel" name="mainLabel">
       <property name="font">
        <font>
         <pointsize>72</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Twoje rezerwacje:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
